### DISCUSSION TITLE
linux/ARM 64 support

### DISCUSSION THREAD

**Discussion by francescouni0** [OpenGATE/opengate#621] [Technical issues] [answered]
Will support for linux arm64 be added any time soon?

**Comment by tbaudier** [comment, marked as answer]
Hello,

For the moment, we provide only x86_64 wheels. For arm64 wheels, it's not planned to do it because nobody asked for before.
And I do not fully understand the docs but it seems arm machines are not available for free plans: https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#overview-of-github-hosted-runners

By the way, you can still install opengate following the developer installation workflow

**Comment by SimonRit** [comment]
I don't think you need a specific runner, ITK compiles them on the same one but with a different manylinux:
https://github.com/InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/commit/7ad0d6448c52715d9035dc8e03bf3b9d18f1a0bd

**Comment by francescouni0** [comment]
Actually, following the installation for developers solved the problem for me. Thank you all !!

**Comment by dsarrut** [comment]
I open an issue, we may look to build a wheel wityh arm64 in the future.
